netty/src/main/java/io/grpc/netty/UdsNettyChannelProvider.java

```diff
@@ -44,47 +41,23 @@ public int priority() {
 
   @Override
   public NettyChannelBuilder builderForAddress(String name, int port) {
-    throw new UnsupportedOperationException("host:port not supported");
+    throw new AssertionError("NettyChannelProvider shadows this implementation");
   }
 
   @Override
   public NettyChannelBuilder builderForTarget(String target) {
-    ChannelCredentials creds = InsecureChannelCredentials.create();
-    ProtocolNegotiators.FromChannelCredentialsResult result = ProtocolNegotiators.from(creds);
-    if (result.error != null) {
-      throw new RuntimeException(result.error);
-    }
-    return getNettyChannelBuilder(target, creds, null, result.negotiator);
+    throw new AssertionError("NettyChannelProvider shadows this implementation");
```

Comment:
Good catch. Realized there is no way to invoke this using the new `ManagedChannelRegistry` API.